Boolean.CSharp.Test/AccountTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Boolean.CSharp.Main;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Boolean.CSharp.Test
+{
+    [TestFixture]
+    public class AccountTests
+    {
+        private Customer _user;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _user = new Customer();
+            var stringWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+        }
+
+        [Test]
+        public void userCanCreateSavingsAndGeneralAccount()
```

Comment:
your tests are good; though they seem to be multiple tests in one single function; probabley better to split them up a bit for clarity